lti_consumer/outcomes.py

```diff
@@ -81,7 +83,7 @@ def parse_grade_xml_body(body):
     return imsx_message_identifier, sourced_id, score, action
 
 
-class OutcomeService(object):
+class OutcomeService:  # pylint: disable=bad-option-value, old-style-class
```

Comment:
Same as above, we should not remove the explicit dependence on (object) just yet because this code will need to run in both python 2 and python3.